protocols/49de51/thermo_magmax_plant_isolation.ot2.apiv2.py

```diff
@@ -70,126 +72,94 @@ def pick_up_plate():
     
     def initial_lysis(plate):
         pick_up_plate()
-        [p300m.transfer(500, lysis_buffer_a_wash_1, col) for col in plate.columns()]
+        [p300m.transfer(500, lysis_buffer_a_wash_1, col) for col in plate_cols]
```

Comment:
A general stylistic note for protocols: list comprehensions are great for variable assignments (i.e. accessing wells of a plate in some order), but they can be problematic for API commands like `transfer`. If we need to troubleshoot later and break the `transfer` into multiple atomic commands, we'll need to change the list comp to a loop